Assets/MixedRealityToolkit.Extensions/QRCodesTracker/Scripts/QRCodesManager.cs

```diff
@@ -0,0 +1,297 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// The process for distributing QRCodesTrackerPlugin.dll is not yet defined. This code will remain unusable
+// to the general public until said distribution story is determiend. However, this file has been added to enable
+// public facing development.
+// #define QRCODESTRACKER_BINARY_AVAILABLE
+
+#if QRCODESTRACKER_BINARY_AVAILABLE
+
+#if UNITY_WSA
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+#if WINDOWS_UWP
+using Windows.Perception.Spatial;
+using Windows.Perception.Spatial.Preview;
+#endif
+
+using QRCodesTrackerPlugin;
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.QRCodesTracker
+{
+    public static class QRCodeEventArgs
+    {
+        public static QRCodeEventArgs<TData> Create<TData>(TData data)
```

Comment:
Why is this method in a different class?

Reply:
Also, why have a Create method that does simple constructor instantiation?

---
In reply to: [275471004](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3937#discussion_r275471004) [](ancestors = 275471004)

Reply:
Will address qrcodesmanager changes in a separate review. This code is currently being worked on in a different repo.